license-maven-plugin/src/main/java/com/mycila/maven/plugin/license/dependencies/LicensePolicy.java

```diff
@@ -51,16 +51,10 @@ public boolean getPredicate() {
      * Simple policy decision based on whether a matcher succeeded.
      *
      * @param matched - boolean result of some matching operation.
-     * @return
+     * @return true, if is allowed
      */
     public boolean isAllowed(final boolean matched) {
-      if (matched && allowed) {
-        return true;
-      } else if (!matched && !allowed) {
-        return true;
-      } else {
-        return false;
-      }
+      return (matched && allowed) || (!matched && !allowed);
```

Comment:
`return !(matched ^ allowed)`

Reply:
think that does not apply given how the checks are but Eclipse is suggesting far, far easier -> 'returned matched == allowed'.

Reply:
updating PR with suggestion from eclipse better look at.

Reply:
lol indeed ;-) `false == false`